pyproject.toml

```diff
@@ -1,13 +1,78 @@
 [build-system]
-requires = [
-    "setuptools<=63",
-    "wheel",
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "zamba"
+version = "2.3.0"
+authors = [
+  {name = "DrivenData", email = "info@drivendata.org"}
+]
+description = "Zamba is a command line tool and Python package to identify animals in camera trap videos and train custom models for new species and habitats."
+readme = "README.md"
+license = {text = "MIT License"}
+classifiers = [
+  "Programming Language :: Python :: 3",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: OS Independent",
+]
+requires-python = ">=3.7"
+dependencies = [
+  "appdirs",
+  "av",
+  "cloudpathlib[s3]",
+  "ffmpeg-python",
+  "future",
+  "fvcore",
+  "gitpython",
+  "importlib_metadata ; python_version < '3.8'",
+  "loguru",
+  "numpy",
+  "opencv-python-headless",
+  "openpyxl",
+  "pandas>1.2.0",
+  "pandas_path",
+  "pqdm",
+  "pydantic",
+  "python-dotenv",
+  "pytorch-lightning>=1.6.0",
+  "pytorchvideo",
+  "scikit-learn",
+  "tensorboard",
+  "thop==0.0.31.post2005241907",
+  "timm",
+  "torch",
+  "torchinfo",
+  "torchvision>=0.10.0",
+  "tqdm",
+  "typer[all]",
+  "yolox @ git+https://github.com/r-b-g-b/YOLOX.git@build-requirements",
+]
+
+[project.scripts]
+zamba = "zamba.cli:app"
+
+[project.optional-dependencies]
+tests = [
+  "coverage",
+  "Pillow>=9.0.0",
+  "pytest",
+  "pytest-coverage",
+  "pytest-mock",
+  "wheel",
 ]
-build-backend = "setuptools.build_meta"
+densepose = [
+  "detectron2-densepose @ git+https://github.com/facebookresearch/detectron2@main#subdirectory=projects/DensePose"
+]
+
+[project.urls]
+homepage = "https://github.com/drivendataorg/zamba"
+bug_tracker = "https://github.com/drivendataorg/zamba/issues"
```

Comment:
In case the aesthetic appearance of these links is something you care about: if you wrap these keys in double-quotes like `"Bug Tracker"` then you can have spaces in the link names.

I found out that these keys do show up as written. Example of a package that wrote out `bug-tracker` lowercase with a hyphen:

https://github.com/wntrblm/nox/blob/119fd00cbb7584127e6deae5c1e4c1c9811f850e/pyproject.toml#L56-L60

https://pypi.org/project/nox/

Reply:
thanks for the example, updated!